application/classes/Ushahidi/Core.php

```diff
@@ -73,6 +73,11 @@ public static function init()
 				and Kohana::$config->load('features.private.enabled');
 		});
 
+		// Intercom config settings
+		$di->set('site.intercomAppToken', function() use ($di) {
+			return Kohana::$config->load('site.intercomAppToken');
```

Comment:
Yea. Its not super clear where it should live. Probably just `thirdparty` or something similar would work. I think you could just call `getenv()` directly here and skip the kohana config entirely